### PR TITLE
ios_cordova_changes

### PR DESCRIPTION
**Pull request by IshmaelPazniak**
Cordova adjustments for iOS

### CODE REVIEW

File: src/ios/CDVInsert.m [hunk 1]
```diff
@@ -73,10 +64,37 @@ - (void)initSDK:(CDVInvokedUrlCommand *)command {
...
-    [[InsertManager sharedManager] initSDK:appKey companyName:companyName];
+    NSDictionary *userAttributes;
+    NSString *visitorId;
+    NSString *accountId;
+    if (command.arguments.count >= 3) {
+        userAttributes = command.arguments[0];
+        visitorId = command.arguments[1];
+        accountId = command.arguments[2];
+    } else {
+        [self sendPluginResult:CDVCommandStatus_OK command:command message:@"Expecting 2 arguments: actionName, params"];
```
Comment (night2tl) [inline src/ios/CDVInsert.m]: Change the text, i.e expecting 3 arguments.

File: src/ios/CDVInsert.m [hunk 2]
```diff
@@ -73,10 +64,37 @@ - (void)initSDK:(CDVInvokedUrlCommand *)command {
...
+    }
+    InsertInitParams *initParams = [[InsertInitParams alloc] init];
+    if (userAttributes)
```
Comment (night2tl) [inline src/ios/CDVInsert.m]: Can you add braces to single if statements (just for the sake of conventions)

File: src/ios/CDVInsert.m (hunk 1 above)
Comment (IshmaelPazniak) [reply src/ios/CDVInsert.m]: Done.

File: src/ios/CDVInsert.m (hunk 2 above)
Comment (IshmaelPazniak) [reply src/ios/CDVInsert.m]: Done.

**Pull request merged**